nx-X11/lib/src/x11_trans.c

```diff
@@ -24,6 +24,10 @@
 #include "config.h"
 #endif
 
+#ifndef MAXHOSTNAMELEN
```

Comment:
> Hmm, isn't that a backport and should be part of a separate commit?

Hmm... I simply fixed the build with this change. No intentional backport, really. In fact, the x11_trans.c file was created by @uli42 in commit d3ae0b2c9559e70448746240caaf5634c78d0eef.